.github/workflows/ci.esp32s3.dev.yml

```
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     name: Build & Test (esp32s3)
 
-    steps:
+    steps:          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)